src/settings/LostItemFeePolicy/components/ViewSections/LostItemFeeSection/LostItemFeeSection.test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { noop } from 'lodash';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import LostItemFeeSection from './LostItemFeeSection';
+
+const testSectionKeyValues = (testName, testId, expectedValue) => {
+  it(`should render ${testName} with ${expectedValue} value`, () => {
+    expect(screen.getByTestId(testId)).toBeVisible();
```

Comment:
looks like we can store `screen.getByTestId(testId)` in variable to reuse it

Reply:
Good idea, agree, thanks